tests/unit_tests/test_ustar_cp/test_cpdBootstrapUStarTh4Season20100901.py

```diff
@@ -0,0 +1,263 @@
+"""Test module for the cpdBootstrapUStarTh4Season20100901 matlab function.
+
+This module contains the unit tests for the cpdBootstrapUStarTh4Season20100901. 
+These tests cover basic behaviour, edge cases and errors.
+"""
+
+import pytest
+import matlab.engine
+import numpy as np
+import json
+import csv
+import os
+
+@pytest.fixture
+def load_json(name):
+    """Loads json."""
+
+    with open(name, 'r') as file:
+        return json.load(file)
+
+@pytest.fixture(scope="module")
+def matlab_engine():
+    # Initialize MATLAB engine
+    eng = matlab.engine.start_matlab()
+    eng.addpath('/home/tcai/Projects/ONEFlux/oneflux_steps/ustar_cp')
```

Comment:
Adding the path isn't necessary and it's hard coded to your personal machine. You can set the path by overriding the default arg in the matlab_engine picture defined in contest.py

Reply:
Resolved